lang/java/reef-tang/tang/src/test/java/org/apache/reef/tang/ClassHierarchyDeserializationTest.java

```diff
@@ -120,15 +132,14 @@ public void testDeserializationForEvent() {
   public void testBindSetEntryWithSetOfT() throws IOException {
     final ClassHierarchy ns1 = Tang.Factory.getTang().getDefaultClassHierarchy();
     ns1.getNode(SetOfClasses.class.getName());
-    final ClassHierarchy ns2 = new ProtocolBufferClassHierarchy(ProtocolBufferClassHierarchy.serialize(ns1));
+    final ClassHierarchy ns2 = classHierarchySerializer.fromString(classHierarchySerializer.toString(ns1));
```

Comment:
Some package-private classes like `SetOfStrings` do not allow moving this class to another package. I think we can keep this test here: once REEF-400 is resolved, then the deserialization can be done regardless of the implementation of `ClassHierarchySerializer`. Does it make sense?